spec/models/teacher_spec.rb

```diff
@@ -32,3 +32,39 @@
     end
   end
 end
+
+require 'irb'
+require 'rails_helper'
+RSpec.describe Teacher, type: :model do
+  describe 'happy path' do
+    it 'returnes valid object' do
+      teacher = create(:teacher, attributes_for(:teacher))
+      expect(teacher).to be_valid
+    end
+  end
```

Comment:
Please avoid writing such tests - you're essentially testing your FactoryBot + Rails setup, no application specific logic :) I can assure you that both Rails and FactoryBot maintain an extensive set of tests to assure they work properly 😄 

However, if you really want to be sure that your factories are alright before the test, you can use `FactoryBot.lint!`, typically as a hook before all tests.